version/generate_version

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# generates the version of file based on current git version
+
+cd $GOPATH/src/github.com/contiv/netplugin/version
+
+USE_RELEASE=$1
+
+if [ -z "${USE_RELEASE}" ]
+then
+  file_name=version_default.go
+  UseVersionStr="// +build !use_version" 
```

Comment:
@mapuri - missed this one.
`use_version` would need to be triggered when doing `make release`; ideally when we start making fixed release tags. For now I will make the release tag consistent with `-version` option output. Will make this fix now.